ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml

```diff
@@ -65,7 +65,7 @@ tests:
   container:
     from: src
 - as: versions
-  commands: INTERACTIVE=false make versions && git diff --exit-code
+  commands: make check-versions
```

Comment:
nice! 
Could you considering improving the commit message 

E.g. 

```
MON-3173: Use new check-versions target to validate versions

This commit uses the new `check-versions` make target to
ensure synchronization of versions. This additionally improves the
readability of logs as they now show the output of `git diff --exit-code`.

```

Reply:
Since the CI passed and I am going to give it 

/lgtm

Reply:
After a discussion with Sunil, I changed the format but kept the wording.